.github/workflows/tests.yml

```diff
@@ -76,11 +79,15 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           cache: pip
-          python-version: "3.11"
+          python-version: "3.10"
```

Comment:
What's broken with Python 3.10?

Reply:
It's not about something being wrong about python version, it's about consistency of built wheels across all the tests, so that when it's combined in coverage report after, it has an access to the correct source. All could be changed to 3.11. Or to 3.10, it doesn't matter as long as it's the same.